app/components/cases/cases-header.hbs

```diff
@@ -1,34 +1,40 @@
-{{#if (not isAddingCase)}}
-  <div class="auk-toolbar-complex auk-toolbar-complex--auto">
-    <div class="auk-toolbar-complex__left">
-      <div class="auk-toolbar-complex__item">
-        <h1 class="auk-toolbar__title" data-test-cases-header-title>
-          {{t "cases"}}
-        </h1>
-      </div>
+<div class="auk-toolbar-complex auk-toolbar-complex--auto">
+  <div class="auk-toolbar-complex__left">
+    <div class="auk-toolbar-complex__item">
+      <h1 class="auk-toolbar__title" data-test-cases-header-title>
+        {{t "cases"}}
+      </h1>
     </div>
+  </div>
+  {{#if currentSession.isEditor}}
```

Comment:
Zorgen dat variabelen die gebruikt worden in de template steeds gescoped zijn met `@` of `this` (tenzij het een lokale template variabele is zoals bvb in een `each`-loop)

Deze comment geldt op nog enkele plaatsen in deze PR.

Reply:
Ik ben nog eens door alle files gegaan en heb er inderdaad enkelen kunnen uithalen.
Het is wel best gemakkelijk erover te lezen bij het overnemen van code.